decoders/connector/volley-boast/vobo-gp-1/v1.0.0/payload.js

```diff
@@ -0,0 +1,1512 @@
+        let stringToPrint = payload.timestamp + " | " + payload.deveui + " | " + payload.data.voboType + " | " + payload.data.payloadType + " | ";
+        for (let idx = 0; idx < dataKeys.length; idx++) {
+            let fieldName = dataKeys[idx];
+            let valueHex = "0x" + payload.data.modbusSlots[fieldName].toString(16).toUpperCase().padStart(4, 0);
+            stringToPrint += fieldName + " = " + valueHex + " | ";
+        }
+        console.log("%s\n", stringToPrint);
+        return;
+    }
+
+    if (payload.data.payloadType == "Digital Inputs") {
+        var stringToPrint = ""
+        for (let i = 0; i < payload.data.digitalSensorData.length; i++) {
+            stringToPrint = stringToPrint + payload.data.digitalSensorStrings[i] + " = " + payload.data.digitalSensorData[i].toString() + " | ";
+        }
+        console.log("%s | %s | %s | %s | %s\n", payload.timestamp, payload.deveui, payload.data.voboType, payload.data.payloadType, stringToPrint);
+        return;
+    }
+
+    if (payload.data.payloadType == "Event Log") {
+        let eventTimestampDateTime = new Date(payload.data.eventTimestamp * 1000).toISOString();
+        let eventCodeHexString = payload.data.eventCode.toString(16).toUpperCase().padStart(4, 0);
+        let metadataHexString = Buffer.from(payload.data.metadata).toString("hex").toUpperCase();
+        console.log("%s | %s | %s | %s | Timestamp = %s | Code = 0x%s | Metadata = 0x%s |\n", payload.timestamp, payload.deveui, payload.data.voboType, payload.data.payloadType, eventTimestampDateTime, eventCodeHexString, metadataHexString);
+        return;
+    }
+
+    if (payload.data.payloadType == "Modbus Generic") {
+        let dataKeys = Object.keys(payload.data);
+        let stringToPrint = payload.timestamp + " | " + payload.deveui + " | " + payload.data.voboType + " | " + payload.data.payloadType + " | ";
+        for (let idx = 0; idx < dataKeys.length; idx++) {
+            if ((dataKeys[idx] != "fport") && (dataKeys[idx] != "voboType") && (dataKeys[idx] != "payloadType")) {
+                let fieldName = dataKeys[idx];
+                let regValueHexStr = "0x" + payload.data[fieldName].toString(16).toUpperCase().padStart(4, 0);
+                stringToPrint += fieldName + " = " + regValueHexStr + " | ";
+            }
+        }
+        console.log("%s\n", stringToPrint);
+        return;
+    }
+
+    console.log(JSON.stringify(payload, null, 4));
+    console.log("");
+}
+
+//===========================================================
+// Changes made to decoder for TagoIO below this point. 
+// TagoIO executes the decoder file (payload.js) rather than
+// calling the Decoder function, so code must be added here 
+// to map the arguments to the Decoder function, and the 
+// exports had to be commented out.
+//===========================================================
+
+const bytes_vb = payload.find((x) => x.variable === "bytes");
```

Comment:
The "bytes" variable should be changed to payload since that is the variable all TagoIO Networks emit.

Reply:
Please replicate in all the connectors

Reply:
Pushed new commit with fixes. Thanks.